.changeset/real-turkeys-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-react': patch
+---
+
+Use debounced handler for submitting
```

Comment:
Hey sorry, the spell checker didn't know this one. You can add the word "debounced" to `.github/vale/config/vocabularies/Backstage/accept.txt` to make it happy.